.betterer.results

```diff
@@ -1032,6 +1010,39 @@ exports[`better eslint`] = {
       [0, 0, 0, "Do not use any type assertions.", "0"],
       [0, 0, 0, "Do not use any type assertions.", "1"]
     ],
+    "packages/grafana-ui/src/graveyard/GraphNG/GraphNG.tsx:5381": [
```

Comment:
sorry... this doubles the betterer errors, but at least half are in a graveyard ☠️ 💀 ☠️

Reply:
@ryantxu what's the plan for `/graveyard`? i'm guessing we remove at some point in the future... what is that point? we could ignore `/graveyard` in betterer (that's what we do for `/angular` for example)